examples/common/src/lib.rs

```diff
@@ -21,7 +21,7 @@ impl Plugin for CommonPlugin {
             AssetsPlugin,
             CorePlugin,
             GamePlugin,
-            EditorPlugin::default(),
+            //EditorPlugin::default(),
```

Comment:
is this supposed to be commented out?

Reply:
Yes ! Sadly, Bevy_editor_pls does not [yet](https://github.com/jakobhellermann/bevy_editor_pls/pull/110) have a Bevy 0.14 compatible release , so I left it commented out for anyone who might really need it for the legacy code at some point.